### PR TITLE
add pawan-giri file

### PR DESCRIPTION
**Pull request by pawankrgiri**
added a branch and put forward pull request

### CODE REVIEW

File: _posts/0000-01-02-pawan-giri.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+---
+layout: slide
+title: "Welcome to our second slide!"
+---
+Your text
```
Comment (pawan-giri) [inline _posts/0000-01-02-pawan-giri.md:5]: *Intelligence is a very valuable thing innit, my friend?
And usually, it comes far too fucking late.*

Comment (pawan-giri) [inline _posts/0000-01-02-pawan-giri.md:5]: it comes far too late

Comment (pawan-giri) [inline _posts/0000-01-02-pawan-giri.md:5]: it comes far too late

Comment (pawan-giri) [inline _posts/0000-01-02-pawan-giri.md:5]: it comes far too late

**Review by pawankrgiri — COMMENTED** [review]
it comes far too late

File: _posts/0000-01-02-pawan-giri.md (hunk 1 above)
Comment (pawan-giri) [inline _posts/0000-01-02-pawan-giri.md:5]: it comes far too late

**Review by pawankrgiri — COMMENTED** [review]
k

File: _posts/0000-01-02-pawan-giri.md (hunk 1 above)
Comment (pawankrgiri) [inline _posts/0000-01-02-pawan-giri.md]: welcome how can i use this

**Review by pawankrgiri — COMMENTED** [review]
ya

File: _posts/0000-01-02-pawan-giri.md (hunk 1 above)
Comment (pawankrgiri) [reply _posts/0000-01-02-pawan-giri.md]: yep

File: _posts/0000-01-02-pawan-giri.md [hunk 2]
```diff
@@ -0,0 +1,6 @@
+---
+layout: slide
+title: "Welcome to our second slide!"
+---
+ok
```
Comment (pawankrgiri) [inline _posts/0000-01-02-pawan-giri.md:5]: fdf

Comment (pawankrgiri) [reply _posts/0000-01-02-pawan-giri.md:5]: cvc

**Pull request merged**